pkg/client/unversioned/kubelet/kubelet_client.go

```diff
@@ -14,39 +14,61 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package unversioned
+package kubeletclient
```

Comment:
also please make the package name match the directory name, wherever it ends up.